lib/utils/schema.js

```diff
@@ -85,6 +85,10 @@ module.exports = {
     author: {
       type: 'string',
       pattern: '\S*',
+    },
+    packageKey: {
```

Comment:
This is a great idea, although I worry that this would encourage people committing their keys into source control. Would it be better to have this pulled from a separate file?

Reply:
I think there is enough flexibility, peoples can add their keys to `ukor.local`, and keep them away from repository, or they can use `ukor.properties,yaml` and pushing the key into a repository, so all team members will have access to the key.

Reply:
In other order, from my point of view, is more handy to have all config files in one place.

Reply:
👍 You're right. The option to have keys in the `ukor.local` file should offer enough flexiblity. LGTM